### ISSUE TITLE
Frontend export without file extension

### ISSUE THREAD

**Issue by Patta** [einpraegsam/powermail#416]
The exported xls or csv file from the frontend list view has no file extension and the file name is just "Download" in chrome. With other browser the filename is "Output" or is cryptically.

The chrome console show this:
`Resource interpreted as Document but transferred with MIME type application/xls: "https://domain.localhost/page-with-mails/?type=31311&tx_powermail_pi2%5Baction%5D=export&tx_powermail_pi2%5Bcontroller%5D=Output&cHash=aefaa3769aff8797fe738e20fdf08d63".`

The xls file can opened with excel and seems correct.

TYPO3 8.7.24
powermail 7.2.0 (tested with default templates and configuration)
PHP 7.3
composer mode

**Comment by einpraegsam** [comment]
ok, thx for your note. Nevertheless I have no clue how to fix this. Any ideas? PR are fine of course.

**Comment by dmnkj** [comment]
I have the same problem. I fixed it temporarily by changing the additional headers in /Powermail_Frontend/setup.txt.

from
additionalHeaders.10.header = Content Type: application / xls | Content Disposition: Inline; Filename = export.xls | Cache control: | pragma:

to
additionalHeaders.10.header = content type: application / xls
additionalHeaders.20.header = Content Disposition: Appendix; Filename = export.xls
additionalHeaders.30.header = Cache Control: No cache
additionalHeaders.40.header = Pragma: No cache

Tested in Firefox, Edge and IExplorer

I do not know if that's the main problem, but it worked for me. Maybe that helps further to solve the problem.
